api/api/specs/routers.yaml

```diff
@@ -737,7 +738,7 @@ components:
       type: "object"
       required:
         - image
-        - resource_request
+        - autoscaling_policy
```

Comment:
Making `resource_request` optional while `autoscaling_policy` becomes required, as detailed in the PR description.